akoshkin/02.CSS/Task.Add.02/Task.Add.03.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html> 
```

Comment:
Do not mix both approach here
or float of flex, not both
Even width you can declarative be flex